include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -773,7 +773,11 @@ typedef struct OrtMIGraphXProviderOptions {
    *   \note If a ::OrtArenaCfg has been applied, it will override this field
    */
   int migraphx_arena_extend_strategy;
-
+  int migraphx_bf16_enable;             // MIGraphX BF16 precision. Default 0 = false, nonzero = true
```

Comment:
This is an ABI change. Could you please avoid making such changes?

Reply:
You were accepting such changes before #23404, #23534, #21599, and #20643.

Reply:
I think the current change is fine in terms of not changing any existing values or data types and appending at the end.

Does the implementation handle a user creating the EP with the old version of the options gracefully? i.e. how do you know if the user is aware of the new fields and is providing them? example use case would be existing user code where the onnxruntime.dll is replaced with a new version. user code doesn't need to be recompiled in this scenario but would only be aware of the original definition of this struct.